src/lib/core/components/computations/PassThroughComputation.tsx

```diff
@@ -41,6 +42,7 @@ const visualizationTypes: Record<string, VisualizationType> = {
   // densityplot: scatterplotVisualization,
   barplot: barplotVisualization,
   boxplot: boxplotVisualization,
+  ['map-markers']: mapVisualization,
```

Comment:
```suggestion
  'map-markers': mapVisualization,
```

Clears up an eslint warning

Reply:
Ooh thanks.  I've used the square brackets before.  Not sure why not this time!

Reply:
eslint just said they're not necessary for this key value. Nothing major 👍